semantic-source/src/Source/Source.hs

```diff
@@ -87,7 +88,7 @@ fromText = Source . T.encodeUtf8
 
 -- | Return the Text contained in the 'Source'.
 toText :: Source -> T.Text
-toText = T.decodeUtf8 . bytes
+toText = T.decodeUtf8With lenientDecode . bytes
```

Comment:
There's no reason for us to fail decoding (and we do see many examples of code files with non-utf8 characters in them). This is an appropriate tradeoff for the kinds of things we use semantic for.